src/domain/individualHarbor/IndividualHarborPageContainer.tsx

```diff
@@ -46,6 +49,29 @@ const IndividualHarborPageContainer: React.SFC = () => {
       accessor: 'identifier',
       filter: 'text',
     },
+    {
+      Cell: ({ cell }: { cell: Cell<Berth> }) => {
+        const activeLease = cell.row.original.leases?.find(
+          lease => lease.isActive && lease.status === LeaseStatus.PAID
```

Comment:
backend already checks the status and only `PAID` could be `isActive` 🙂

Reply:
Ofcourse! Fixed.